Banyan/src/sessions.jl

```diff
@@ -137,6 +140,7 @@ function start_session(;
                 s3_put(get_aws_config(), s3_bucket_name, "$(environment_hash)/Manifest.toml", manifest_toml)
             end
         end
+    # TODO: Else case is the same
```

Comment:
I think you can take this comment off, since this is no longer a TODO!

Reply:
Yup